physionet-django/project/views.py

```diff
@@ -2307,3 +2314,33 @@ def generate_signed_url(request, project_slug):
     GCPLog.objects.update_or_create(project=project, user=request.user, data=data)
 
     return JsonResponse({'url': url})
+
+
+class PublishedProjectList(mixins.ListModelMixin, generics.GenericAPIView):
+    """
+    List all Published Projects
+    """
+    queryset = PublishedProject.objects.all().order_by('id')
+    authentication_classes = [SessionAuthentication, BasicAuthentication]
+    # permission_classes = [IsAuthenticated]
+    serializer_class = PublishedProjectSerializer
+
+    def get(self, request, *args, **kwargs):
+        return self.list(request, *args, **kwargs)
+
+
+class PublishedProjectDetail(mixins.RetrieveModelMixin, generics.GenericAPIView):
+    """
+    Retrieve an Published Project
+    """
+
+    authentication_classes = [SessionAuthentication, BasicAuthentication]
+    # permission_classes = [IsAuthenticated]
```

Comment:
Perhaps remove the `# permission_classes = [IsAuthenticated]` comments in this file?